python/revng/model/migrations/migrate.py

```diff
@@ -0,0 +1,41 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from revng.model import latest_version  # type: ignore
+
+from revng.model.migrations.versions.v2 import Migrate2
+from revng.model.migrations.versions.v3 import Migrate3
+from .migration import Migration
+
+migrations: dict[int, Migration] = {
+    2: Migrate2(),
+    3: Migrate3(),
+}
+
+
+def migrate(model: dict):
+    if "Version" not in model:
+        return
+
+    version = model["Version"]
+
+    if type(version) is not int:
```

Comment:
Use `isinstance` to check the type of something.

Reply:
Done.

Reply:
Actually, I would not perform those checks. We'd need a JSON schema for the schema to perform those kind of checks.

I'd avoid "littering" the codebase with checks that belong to JSON schema. Leave a TODO about having JSON schema and drop the checks (or just `assert` if you really want to).

Reply:
Done.